packages/web-api/src/types/request/assistant.ts

```diff
@@ -0,0 +1,26 @@
+import type { TokenOverridable } from './common';
+
+// https://api.slack.com/methods/assistant.threads.setStatus
+export interface AssistantThreadsSetStatusArguments extends TokenOverridable {
+  channel_id: string;
+  status: string;
+  thread_ts: string;
+}
+
+// https://api.slack.com/methods/assistant.threads.setSuggestedPrompts
+export interface AssistantThreadsSetSuggestedPromptsArguments extends TokenOverridable {
+  channel_id: string;
+  prompts: {
```

Comment:
Are 0 prompts acceptable to send in the request? If not, I type arguments that are of the form "at least one" like so:

```typescript
prompts: [SomeType, ...SomeType[]];
```

So if at least 1 prompt must be provided, I'd factor the prompt object w/ `title` and `message` into its own interface and use the above technique to signal to developers "at least 1." See for example [this part of `reminders.ts`](https://github.com/slackapi/node-slack-sdk/blob/main/packages/web-api/src/types/request/reminders.ts#L13).

Reply:
Great suggestion – added and confirmed it works as expected in a sample.